visualization/app/codeCharta/state/store/appSettings/amountOfTopLabels/amountOfTopLabels.actions.ts

```diff
@@ -18,4 +18,4 @@ export function setAmountOfTopLabels(amountOfTopLabels: number = defaultAmountOf
 	}
 }
 
-export const defaultAmountOfTopLabels = 1
+export const defaultAmountOfTopLabels = 10
```

Comment:
Please use something like `Math.min(Math.floor(numberOfBuildings / 100), 10))`. That way it's not a static value as a lot of labels for very small projects would be quite an overhead.